cmd/oras/root/manifest/index/update.go

```diff
@@ -80,6 +79,10 @@ Example - Update an index and output the index to stdout, auto push will be disa
 				return err
 			}
 			opts.RawReference = args[0]
+			// ignore --pretty when output to a file
+			if opts.outputPath != "" && opts.outputPath != "-" {
+				opts.Pretty.Pretty = false
+			}
```

Comment:
Can be merged into the handler construction.

Reply:
If this is a general option thing, maybe a method in otpions?

Reply:
> If this is a general option thing, maybe a method in otpions?

@TerryHowe We can move it to content handler to avoid code repeat.